### PR TITLE
Replace Big Data Cluster with big data cluster

### PR DESCRIPTION
**Pull request by kevcunnane**
Official docs guidance is to use "big data cluster" instead of "Big Data Cluster"

### CODE REVIEW

**Comment by Charles-Gagnon** [comment]
Remind me again why we're using double quotes in these scenarios? Is it just a convention thing to align with other languages?

File: resources/xlf/en/big-data-cluster.xlf [hunk 1]
```diff
@@ -24,7 +24,7 @@
   </body></file>
   <file original="extensions/big-data-cluster/out/bigDataCluster/tree/addControllerTreeNode" source-language="en" datatype="plaintext"><body>
     <trans-unit id="textBigDataClusterControllerWithDots">
-      <source xml:lang="en">Add Big Data Cluster Controller...</source>
+      <source xml:lang="en">Add big data cluster Controller...</source>
```
Comment (Charles-Gagnon) [inline resources/xlf/en/big-data-cluster.xlf]: You lowercased controller earlier - did you want to do that here too?

File: resources/xlf/es/big-data-cluster.es.xlf [hunk 2]
```diff
@@ -35,8 +35,8 @@
   <file original="extensions/big-data-cluster/out/bigDataCluster/tree/addControllerTreeNode" source-language="en" target-language="es" datatype="plaintext">
     <body>
       <trans-unit id="textBigDataClusterControllerWithDots">
-        <source xml:lang="en">Add Big Data Cluster Controller...</source>
-        <target state="new">Add Big Data Cluster Controller...</target>
+        <source xml:lang="en">Add big data cluster Controller...</source>
+        <target state="new">Add big data cluster Controller...</target>
```
Comment (Charles-Gagnon) [inline resources/xlf/es/big-data-cluster.es.xlf]: Same here - lowercase controller?

File: resources/xlf/en/big-data-cluster.xlf (hunk 1 above)
Comment (Charles-Gagnon) [reply resources/xlf/en/big-data-cluster.xlf]: (There's others to update too if you do)

File: resources/xlf/it/mssql.it.xlf [hunk 3]
```diff
@@ -1023,7 +1023,7 @@
         <target state="translated">Il file locale {0} non esiste.</target>
       </trans-unit>
       <trans-unit id="sparkJobSubmission">
-        <source xml:lang="en">No Sql Server Big Data Cluster found.</source>
+        <source xml:lang="en">No Sql Server big data cluster found.</source>
         <target state="translated">Non è stato trovato alcun cluster Big Data di SQL Server.</target>
```
Comment (Charles-Gagnon) [inline resources/xlf/it/mssql.it.xlf]: So a couple of things :

1. Do we actually need to update each separate xlf file or is just updating the English text in the ts file enough to then have the translators see that it's been changed and update the strings accordingly?
2. Do we need to inform the loc team that these have changed or will they do that automatically?
3. I see here that the loc version is "cluster Big Data" which doesn't match the casing there before (Big Data Cluster). Do we need to be really careful for product/feature names and make sure to add a comment to the localize call to tell them that this is a product/feature name and should be treated specially?

File: resources/xlf/en/big-data-cluster.xlf (hunk 1 above)
Comment (kevcunnane) [reply resources/xlf/en/big-data-cluster.xlf]: Yeah I'll fix. thanks for catching

File: resources/xlf/it/mssql.it.xlf (hunk 3 above)
Comment (kevcunnane) [reply resources/xlf/it/mssql.it.xlf]: I'm doing a find & replace all. I believe this is picked up automatically, but not sure?

**Comment by kevcunnane** [comment]
@Charles-Gagnon regarding double-quotes I do it because Anthony & Chris bug me about it. I believe the official reason it it makes it easier to detect localizable strings in the repo, but I don't know what tools do that. Just that I'm meant to fix this.

Long term we should just write a hygiene rule so as we update files we auto-fix this.

**Pull request merged**